__tests__/pullAuthors.test.ts

```diff
@@ -32,19 +32,25 @@ function mockWith(hasGitHubAccount = true, hasTooManyCommits = false) {
             }
         }
     });
+    const edges: any = [];
+    for (let i = 0; i < numCommits; i++) {
+        edges.push({
+            node: {
+                commit: commit
+            }
+        });
+    }
     const graphqlSpy = jest.spyOn(mockGitHub, 'graphql')
-        .mockImplementation(async (query) => ({
+        .mockImplementation(async (query, parameters) => ({
             repository: {
                 pullRequest: {
                     commits: {
-                        totalCount: hasTooManyCommits ? 102 : 1,
-                        edges: [
-                            {
-                                node: {
-                                    commit: commit
-                                }
-                            }
-                        ]
+                        totalCount: numCommits,
+                        edges,
+                        pageInfo: {
+                            endCursor: 'cursor',
+                            hasNextPage: numCommits == 1 || parameters && parameters.cursor ? false : true
```

Comment:
This mock is pretty... janky. It only works for the two test cases we have here: a 2-page query with between 100-200 commits (150 in our case), or a single-commit query.

I had trouble updating these tests due to type failures that I don't fully understand, so I was trying to minimize the changes needed to get them to pass, while also covering the new behaviour.